src/member-api.js

```diff
@@ -149,6 +149,8 @@ export class MemberApi extends TypedEmitter {
           return inviteResponse.decision
         case InviteResponse_Decision.UNRECOGNIZED:
           return InviteResponse_Decision.REJECT
+        case InviteResponse_Decision.DECISION_UNSPECIFIED:
+          return InviteResponse_Decision.REJECT
```

Comment:
Nitpick to save one line:

```suggestion
        case InviteResponse_Decision.UNRECOGNIZED:
        case InviteResponse_Decision.DECISION_UNSPECIFIED:
           return InviteResponse_Decision.REJECT
```